src/cli/cli_tcp.cpp

```diff
@@ -908,6 +944,25 @@ void TcpExample::CompleteBenchmark(void)
 }
 
 #if OPENTHREAD_CONFIG_TLS_ENABLE
+void TcpExample::PrepareTLSHandshake(void)
```

Comment:
We generally treat abbreviation as a name and use `CamelCase` for them (e.g. `MleRouter`, `TcpExample` :)).
```suggestion
void TcpExample::PrepareTlsHandshake(void)
```

Reply:
done